src/name/bizna/ocmos/MMU.java

```diff
@@ -835,17 +835,14 @@ else if((codeUnitBuf[0] & 0xF8) == 0xF0) {
 			if(charToOutput != null) {
 				if(!didOutputChar) {
 					simpleVoidCall(mappedGPU, "set", cursorX, cursorY, charToOutput);
-					didOutputChar = true;
+					didOutputChar = true; // Still set to false by line 846 anyway?
```

Comment:
I think `didOutputChar` has something to do with the logic that suppresses a newline outputted immediately after an automatic linebreak.